file2.js

```diff
@@ -0,0 +1,558 @@
+      $orgBox.selectpicker("refresh");
+    };
+
+    ProjectSettings.prototype.bindUpdateProjectName = function(projectId) {
+      $(".BtnUpdateProjectName").click(function() {
+        var url = $(this).data("url");
+        var postVariable = {};
+        postVariable.projectName = $(".TxtUpdateProjectName").val();
+        postVariable.projectId = projectId.toString();
+
+        utils.JSend(url, postVariable, function(data) {
+          $(".ProjectNameFeedback").showSuccess("Saved");
+          location.reload();
+        }, function(error) {
+          $(".ProjectNameFeedback").showError(error);
+        });
+      });
+
+      $(".TxtUpdateProjectName").keyup(function() {
+        $(".BtnUpdateProjectName").removeClass("disabled");
+      });
+    };
+
+    ProjectSettings.prototype.bindUpdateProjectUrl = function(projectId) {
+      $(".BtnUpdateProjectUrl").click(function() {
+        var projectUrlFeedback = $(".ProjectUrlFeedback");
+        var url = $(this).data("url");
+
+        var projectUrl = $(".TxtUpdateProjectUrl").val();
+        var isPrivate = $(this).data("is-private");
+        if (!UrlUtils.getInstance().isValidUrl(projectUrl, isPrivate)) {
+          projectUrlFeedback.showError("The url is invalid");
+          return false;
+        }
+
+        var postVariable = {};
+        postVariable.projectUrl = projectUrl;
+        postVariable.projectId = projectId.toString();
+
+        utils.post(url, postVariable, function(e) {
+          if (e.error) {
+            projectUrlFeedback.showError(e.error);
+          } else {
+            projectUrlFeedback.showSuccess("Saved");
+            location.reload();
+          }
+        }, function() {
+          projectUrlFeedback.showError("Could not change the url, please try again.");
+        });
+      });
+
+      $(".TxtUpdateProjectUrl").keyup(function() {
+        $(".BtnUpdateProjectUrl").removeClass("disabled");
+      });
+    };
+
+    ProjectSettings.prototype.createFileTree = function(jsonData) {
+      var glyph_opts = {
+        map: {
+          checkbox: "fa fa-square-o",
+          checkboxSelected: "fa fa-check",
+          checkboxUnknown: "fa fa-minus-square",
+          dragHelper: "fa fa-arrow-right",
+          dropMarker: "fa fa-long-arrow-right",
+          error: "fa fa-warning",
+          expanderClosed: "fa fa-caret-right",
+          expanderLazy: "fa fa-angle-right",
+          expanderOpen: "fa fa-caret-down",
+          nodata: "fa fa-meh-o",
+          loading: "fa fa-spinner fa-pulse",
+          // Default node icons.
+          // (Use tree.options.icon callback to define custom icons based on node data)
+          doc: "fa fa-file-o",
+          docOpen: "fa fa-file-o",
+          folder: "fa fa-folder",
+          folderOpen: "fa fa-folder-open"
+        }
+      };
+      $(".file-tree").fancytree({
+        checkbox: true,
+        extensions: ["edit", "glyph"],
+        glyph: glyph_opts,
```

Comment:
![Codacy](https://app.dev.codacy.org/assets/images/favicon.png) Issue found: [Identifier 'glyph_opts' is not in camel case. (camelcase)](https://app.dev.codacy.org/app/codacytestuser/CheckPrCommentsPublicProject/pullRequest?prid=8021)